### DISCUSSION TITLE
Support EventSource

### DISCUSSION THREAD

**Discussion by alexec** [gavv/httpexpect#145] [Ideas]
We use EventSource in our codebase. I imagine supporting this would be similar to WebSocket? #50

**Comment by gavv** [comment]
I guess yes. Likely it would be even simpler to implement because it's unidirectional and because chunked encoding is already working.

**Comment by soggycactus** [comment]
Hey @gavv I'm looking for some OSS work - would it be cool if I picked this up? (Assuming it hasn't been done already)

**Comment by gavv** [comment]
@soggycactus hi, you're welcome! It's not done yet.

**Comment by fer-correa** [comment]
Hi @gavv , This issue still alive? 😄 

Is someone working on it??? Can I try to pick?

**Comment by alexec** [comment]
Could help:

https://github.com/argoproj/argo-workflows/blob/832ae296527bdd51e0c785cdea33ff24a966eb65/test/e2e/argo_server_test.go#L1074

**Comment by soggycactus** [comment]
> Hi @gavv , This issue still alive? 😄
> 
> Is someone working on it??? Can I try to pick?

I ended up not working on this as my project switched to a different protocol, feel free to pick it up

**Comment by fer-correa** [comment]
> > Hi @gavv , This issue still alive? 😄
> > Is someone working on it??? Can I try to pick?
> 
> I ended up not working on this as my project switched to a different protocol, feel free to pick it up

Thanks @soggycactus

**Comment by gavv** [comment]
I've created new issue: https://github.com/gavv/httpexpect/issues/246
